common.py

```diff
@@ -9,262 +9,301 @@
 import discord
 from discord.ext import commands, tasks
 
+invites = {
+    # TTS bot
+    "513423712582762502": "https://discordapp.com/api/oauth2/authorize?client_id=513423712582762502&permissions=36719617&scope=bot",
+    # F@H
+    "565820959089754119": "https://discordapp.com/api/oauth2/authorize?client_id=565820959089754119&permissions=84992&scope=bot",
+    # Channel maker
+    "689564772512825363": "https://discordapp.com/api/oauth2/authorize?client_id=689564772512825363&permissions=68624&scope=bot",
+
+    # TTS bot dev for testing
+    "698218518335848538": "https://example.com"
+}
+
+colours = {
```

Comment:
don't complain about my spelling its spelt correctly I don't care about the other more "correct" way to spell it